site/content/docs/5.0/components/tags.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: docs
+title: Tags
+description: Documentation and examples for tag component.
+group: components
+toc: true
+---
+
+## Standard
+
+{{< example >}}
```

Comment:
Maybe it is something from your tests but I think you should have only the first tag in this example no?